policyengine_us/parameters/gov/irs/payroll/disability_income_exclusion/amount.yaml

```diff
@@ -0,0 +1,13 @@
+description: IRS reduces the excludable disability income by this amount.
+metadata:
+  unit: currency-USD
+  period: year
+  label: Disability income reduction amount 
+  reference:
+    - title: 26 U.S. Code § 105 (d)(3) 
+      href: https://tile.loc.gov/storage-services/service/ll/uscode/uscode1982-01002/uscode1982-010026001/uscode1982-010026001.pdf#page=209
+    # 26 U.S. Code § 105 (d)(3) was repealed in 1983, but some states still uses it as a standard when computing disability income exclusion 
+    - title: Code of the District of Columbia § 47–1803.02. (a)(2)(M)
+      href: https://code.dccouncil.gov/us/dc/council/code/sections/47-1803.02#(a)(2)(M)
+values:
+  2021-01-01: 15_000
```

Comment:
can you find the year it was enacted? this must be a year pre-1983

Reply:
![image](https://github.com/PolicyEngine/policyengine-us/assets/113824411/1c6a5aa5-ee12-42d3-a5d8-6acd9eda6ec4)
[link](https://www.law.cornell.edu/uscode/text/26/105)
Doesn't this mean it was enacted in 1983? I also found this [file (c)(4),(d)(1)](https://www.govinfo.gov/content/pkg/STATUTE-97/pdf/STATUTE-97-Pg65.pdf#page=23)

Reply:
It was repealed in 1983

Reply:
Sorry, the law was enacted in 1976, I have updated the legal code reference.